index.d.ts

```diff
@@ -58,10 +58,11 @@ export interface AnyAction extends Action {
  * @template S The type of state consumed and produced by this reducer.
  * @template A The type of actions the reducer can potentially respond to.
  */
-export type Reducer<S = any, A extends Action = AnyAction> = (
-  state: S | undefined,
-  action: A
-) => S
+export type Reducer<
+  S = any,
+  A extends Action = AnyAction,
+  R extends unknown[] = []
```

Comment:
out of curiosity - does using unknown here in this context gives anything over good old any? tried few things in TS playground and they seem pretty much the same in this regard because R is going to be widened as much as possible anyway and everything extends uknown (and any)

Reply:
Not sure it does in this case. I always just use `unknown` because it's safer. I've used `any` in the past and it's leaked into places I didn't want it to!